contracts/vesting/VestedZeroNFT.sol

```diff
@@ -23,14 +27,19 @@ import {AccessControlEnumerableUpgradeable} from "@openzeppelin/contracts-upgrad
 
 /// @title VestedZeroNFT is a NFT based contract to hold all the user vests
 /// @author Deadshot Ryker <ryker@zerolend.xyz>
-/// @notice NFTs can be traded on secondary marketplaces like Opensea, can be split into smaller chunks to allow for smaller otc deals to happen in secondary markets
+/// @notice NFTs can be traded on secondary marketplaces like Opensea, can be split into smaller chunks 
+/// to allow for smaller otc deals to happen in secondary markets
 contract VestedZeroNFT is
     IVestedZeroNFT,
     AccessControlEnumerableUpgradeable,
     ReentrancyGuardUpgradeable,
     PausableUpgradeable,
     ERC721EnumerableUpgradeable
 {
+	uint256 public constant QUART = 25000; //  25%
```

Comment:
Do we need these variables? Also since we are using proxies for the code; we should ideally set these variables in the initializer

Reply:
Radiant did it this way so I followed them. We can definately move them if you say.